docker-orchestration/osrm-backend/docker-entrypoint.sh

```diff
@@ -8,6 +8,9 @@ PBF_FILE_SUFFIX=".osm.pbf"
 SNAPPY_SUFFIX=".snappy"
 WAYID2NODEIDS_MAPPING_FILE=wayid2nodeids.csv
 NODES2WAY_DB_FILE="nodes2way.db"
+INPUT_STATION_DATA_JSON_FORMAT=input.json
+MOUNT_PATH=/workspace/mnt # mounted host folder
+OASIS_DATA_RELATIVE_PATH="oasisdata"
```

Comment:
Better to be `oasis-data` by convention.

Reply:
ok, copy that